code/lib/blocks/src/controls/Boolean.tsx

```diff
@@ -71,8 +71,11 @@ const Label = styled.label(({ theme }) => ({
   },
 
   'input:checked ~ span:last-of-type, input:not(:checked) ~ span:first-of-type': {
-    background: theme.background.app,
-    boxShadow: `${opacify(0.1, theme.appBorderColor)} 0 0 2px`,
+    background: theme.boolean.selectedBackground,
```

Comment:
seems like there are two conventions going on,

when I saw all the text prefixes like textMutedColor, textColor, etc I had an urge to comment that it could be text.muted etc.

I also see ones like positiveText, warningText etc. so maybe there are 3 naming conventions.

I know these came before this PR but just highlighting it in case this is an easy-ish time to standardize some

Reply:
forgot about this specific line - it could be booleanSelectedBg or booleanSelectedBackground

Reply:
I am hesitant to change the names of existing variables as I don't want to break user themes that rely on these names. I am open to suggestions on preferred naming schemes. If we do decide to rename them (breaking change), now is the time.